<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfWidget</class>
 <widget class="QWidget" name="ConfWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>DParticle Spec</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>LeftSL:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_spinBoxLeftSL">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>TopSL:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_spinBoxTopSL">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>AreaWidth:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="m_spinBoxAreaWidth">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>AreaHeight:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="m_spinBoxAreaHeight">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>DeadUnit:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="m_spinBoxDeadUnit">
            <property name="maximum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>(1:使用百分比  0：使用绝对差）</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>DeadSpec:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="m_spinBoxDeadSpec">
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>DeadType:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="m_spinBoxDeadType">
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>RightSL:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="m_spinBoxRightSL">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>BottomSL:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_spinBoxBottomSL">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Exposure:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="m_spinBoxExposure">
              <property name="maximum">
               <number>65536</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Gain:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="m_spinBoxGain">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="m_checkBoxSaveImage">
              <property name="text">
               <string>SaveImage</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>m_spinBoxLeftSL</zorder>
       <zorder>m_spinBoxRightSL</zorder>
       <zorder>label</zorder>
       <zorder>label_2</zorder>
       <zorder>label_5</zorder>
       <zorder>label_6</zorder>
       <zorder>m_spinBoxAreaHeight</zorder>
       <zorder>m_spinBoxBottomSL</zorder>
       <zorder>label_7</zorder>
       <zorder>label_8</zorder>
       <zorder>label_9</zorder>
       <zorder>label_10</zorder>
       <zorder>label_11</zorder>
       <zorder>label_12</zorder>
       <zorder>label_13</zorder>
       <zorder>m_spinBoxAreaWidth</zorder>
       <zorder>m_spinBoxTopSL</zorder>
       <zorder>m_spinBoxDeadUnit</zorder>
       <zorder>m_spinBoxDeadType</zorder>
       <zorder>m_spinBoxDeadSpec</zorder>
       <zorder>m_spinBoxExposure</zorder>
       <zorder>m_spinBoxGain</zorder>
       <zorder>label_14</zorder>
       <zorder>m_checkBoxSaveImage</zorder>
       <zorder>verticalSpacer</zorder>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="m_actionFollowTheValue">
   <property name="text">
    <string>Follow the value</string>
   </property>
   <property name="toolTip">
    <string>Follow the value</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
